README.md

```diff
@@ -1,10 +1,9 @@
-# MARC Liberation
+# Bibdata
 
-Liberate MARC data from Voyager.
+Formerly known as MARC Liberation (since it liberates MARC data from Voyager).
 
 [![CircleCI](https://circleci.com/gh/pulibrary/bibdata.svg?style=svg)](https://circleci.com/gh/pulibrary/bibdata)
-[![CoverageStatus](https://coveralls.io/repos/github/pulibrary/bibdata/badge.svg?branch=main)](https://coveralls.io/github/pulibrary/bibdata?branch=main)
```

Comment:
@escowles this is used. I added a new coveralls token in circleci and a new repo in coveralls.

Reply:
@christinach 👍🏻 I'll add it back